FrontEnd/src/components/Login/Login.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Grid, Typography } from '@mui/material';
+import image from './login.png';
+import Link from '@mui/material/Link';
+import GetUser, { clientPrincipal } from '../GetUser';
+import { useNavigate } from 'react-router-dom';
+
+export interface LoginPureProps {}
+
+/**
+ * LoginPure welcomes people, and links to /login
+ *
+ * The presentational part of the Login component.
+ */
+export function LoginPure() {
+  return (
+    <Grid container spacing={2}>
+      <Grid item xs={4}>
+        <img
+          src={image}
+          alt="ForestGeo Logo"
+          style={{
+            maxWidth: '100%', // responsive image
+            height: 'auto',
+            padding: 0,
+            margin: 0,
+          }}
+        />
+      </Grid>
+      <Grid item xs={8}>
+        <Typography variant="h3" component="h1">
+          Welcome to the ForestGEO web-app
+        </Typography>
+        <Link href={`/login`}>
```

Comment:
Only thing is, would this make more sense as a button? Otherwise, this all looks great and is a clear example of the steps that need to be taken in order to refactor the rest of the components

Reply:
The way I thought it should be a link rather than a button is: it navigates to a /login URL rather than doing some sort of action. But also... it's the main call to action on the view... so maybe a button would be more appropriate. hrmm.

Reply:
We decided to go with the Button.